Library/Homebrew/extend/ARGV.rb

```diff
@@ -120,6 +120,10 @@ def dry_run?
     include?("--dry-run") || switch?("n")
   end
 
+  def keep_files?
+    flag? "--keep-files"
+  end
```

Comment:
Mike proposed moving this to `ARGV` (see now hidden code comments) and I agree because like Mike I'd like to see this also being used by `brew install`. But good point on using `include?` instead of `flag?`!